### ISSUE TITLE
Some problems about align_lfw.py 

### ISSUE THREAD

**Issue by Carbord** [Joker316701882/Additive-Margin-Softmax#21]
As is mentioned is the README.md, I should align both the training set and lfw. Is this mean that I should use the align_lfw.py for twice? My setting is like following:
Step1 --"python align_lfw.py --input-dir /DataSet/lfwdataset/lfw-deepfunneled --output-dir /home/myname/lfwdataset"
Step2 --"python train.py --data_dir /home/myname/lfwdataset --random_flip --learning_rate -1 --learning_rate_schedule_file ./data/learning_rate_AM_softmax.txt --lfw_dir /home/myrname/lfwdataset --keep_probability 0.8 --weight_decay 5e-4"
Is there any problems? Hope for your reply soon, thank you~

**Comment by Joker316701882** [comment]
@Carbord 
Well, you can't train on lfw while testing on it. Try to train on MS-celeb-1M or VGGface2.
